bikeshed/update/updateCrossRefs.py

```diff
@@ -30,7 +34,7 @@ def update(path, dryRun=False):
         lastMsgTime = config.doEvery(
             s=5,
             lastTime=lastMsgTime,
-            action=lambda: say(f"Downloading data for spec {i}/{len(rawSpecData)}..."),
+            action=progress(i, len(rawSpecData)),
```

Comment:
This is wrong - it's now eagerly evaluating the function (and thus passing `action=None`), rather than producing a lambda. I'll fix.

Reply:
Ah, and I was wondering about this, because I know this code already works correctly; I'd have seen it if the closure was reusing bindings. Technically it is, but the lambda is immediately executed by the `doEvery()` and then thrown away, so it doesn't matter; this is a false-positive error.

That said, eh, it's fine to fix. (And also, I'm so glad JS got this *right* and establishes a fresh binding on every iteration of a for-of loop.)